README.md

```diff
@@ -31,9 +31,9 @@ expect(queue.add.length).to.equal(1);
 
 ## How does it work?
 
-mdn.io uses Google's "I'm feeling lucky" functionality to redirect you to the first search result.
+mdn.io uses DuckDuckGo's "I'm feeling lucky" functionality to redirect you to the first search result.
```

Comment:
Is it called `I'm feeling lucky` over at DDG?

Reply:
It isn't.

I've pushed a fixup commit.

Once we're happy with this branch, I'll rebase it (or you can squash it), and it's like it never happened :D

Reply:
LGTM!